src/casm/mapping/StructureMapping.cc

```diff
@@ -92,9 +92,11 @@ xtal::SimpleStructure make_mapped_structure(
 
   // make the parent structure from the prim for the ideal lattice
   // and site coordinates - occupants are set to "Va"
-  xtal::SimpleStructure parent_structure;
-  parent_structure.lat_column_mat = prim.lattice().lat_column_mat();
-  parent_structure.atom_info.resize(prim.basis().size());
+  xtal::SimpleStructure parent_structure = make_simple_structure(prim);
```

Comment:
I think we should avoid using `make_simple_structure` here in favor of just copying the prim basis site coordinates. I'd like this method to be clear that is only dealing with isotropic atoms, strains, and displacements, and not brining in any properties other than those brought in from `unmapped_structure`. It is hard to think through all the effects that `make_simple_structure` might have, so even if there are checks before and after I think it's easier to skip it.